### ISSUE TITLE
How to output the IOU of the model detection

### ISSUE THREAD

**Issue by 15826694153** [open-mmlab/mmdetection#4340]
In the default output, only the loss_bbox test results, I want to see the iou results of the model detection, how to print the detected IOU in the test results. I am very grateful to you for your help.

**Comment by hhaAndroid** [comment]
You mean is to show iou during the training stage? Can you describe it in detail?

**Comment by 15826694153** [comment]
Thank you very much for your reply.I want to crop out the target object in the picture, so I need to get the accurate detection box.In order to know the accuracy of the detection box, I need to know the IOU score of the model.(There's only one category of my data)

**Comment by kellenf** [comment]
It need change some code, and it's imposible to write in mmdet, because it's hard code. this is my guide for you
import `mmdetection.mmdet.core.bbox.iou_calculators.iou2d_calculator.bbox_overlaps` in  detect_head, and compute iou, and then add hard code in loss function.

**Comment by 15826694153** [comment]
Thank you for your guidance.I need to try.

**Comment by yzk-lab** [comment]
i  

> It need change some code, and it's imposible to write in mmdet, because it's hard code. this is my guide for you
> import `mmdetection.mmdet.core.bbox.iou_calculators.iou2d_calculator.bbox_overlaps` in detect_head, and compute iou, and then add hard code in loss function.
> 
> You need write the code by yourself, good luck to you

i import the mmdetection.mmdet.core.bbox.iou_calculators.iou2d_calculator.bbox_overlaps into the detect head.but i can not control the gt_boxes and anchor ,if i define a function to calculate the iou ,how can i  get the parameter(anchor,gt_bboxes),and then return the iou to the loss functions.because the parameters in loss are pred and target .this puzzles me ,i need your guide,thanks.
